src/lib/tabs/tab-group.spec.ts

```diff
@@ -136,6 +139,53 @@ describe('MdTabGroup', () => {
       expect(component.handleSelection).toHaveBeenCalledTimes(1);
       expect(component.selectEvent.index).toBe(2);
     }));
+
+    it('should update tab positions and attach content when selected', fakeAsync(() => {
+      fixture.detectChanges();
+      let tabComponent = fixture.debugElement.query(By.css('md-tab-group')).componentInstance;
```

Comment:
Out of curiosity, why is this one `let` while the other is `const`? Seems like they could both be `const`.